Sources/ZcashLightClientKit/Initializer.swift

```diff
@@ -433,7 +433,9 @@ public class Initializer {
         if let seed, try await rustBackend.listAccounts().isEmpty {
             var chainTip: UInt32?
             
-            if walletMode == .restoreWallet, let latestBlockHeight = try? await lightWalletService.latestBlockHeight() {
+            // ServiceMode to resolve
+            // called when client starts and restore of the wallet is in progress
+            if walletMode == .restoreWallet, let latestBlockHeight = try? await lightWalletService.latestBlockHeight(mode: .defaultTor) {
```

Comment:
This is a one-off query:
```suggestion
            if walletMode == .restoreWallet, let latestBlockHeight = try? await lightWalletService.latestBlockHeight(mode: .uniqueTor) {
```